cns/NetworkContainerContract.go

```diff
@@ -441,7 +441,7 @@ type GetIPAddressStatusResponse struct {
 
 // GetPodContextResponse is used in CNS Client debug mode to get mapping of Orchestrator Context to Pod IP UUID
 type GetPodContextResponse struct {
-	PodContext map[string]string
+	PodContext map[string][]string
```

Comment:
please add a comment explaining why there can be one orch context to many pod IPs